src/hooks/useFetchData.tsx

```diff
@@ -1,21 +1,24 @@
 import { fetchAPI_json_server } from "../api/fetchAPI_json_server";
 import { useQuery } from "react-query";
 import { fetchAPI_local_statefiles } from "../api/fetchAPI_local_statefiles";
+import { connector } from "../api/connector";
+import { fetchAPI_mongoDB } from "../api/fetchAPI_mongoDB";
 
-export const api_local = new fetchAPI_local_statefiles();
 
+export const connector_mongo = new fetchAPI_mongoDB("http://127.0.0.1:8000/api/v1");
```

Comment:
we should instantiate the fetchApi_local/rest depending on manifest.backendConfig, and use the configured URL for the Rest interface